docs/pages/admin-guides/access-controls/compliance-frameworks/fedramp.mdx

```diff
@@ -132,11 +132,11 @@ teleport:
   #
   # We recommend to use tools like `pwgen` to generate sufficiently random
   # tokens of 32+ byte length.
-  # you can also use auth server's IP, i.e. "10.1.1.10:3025"
+  # you can also use Auth Service instances IP, i.e. "10.1.1.10:3025"
```

Comment:
Not sure if it's in the scope of this PR, but this comment seems to be wrong. Everything except the last line should be a comment for `auth_token`, not `auth_server`.

It looks like this comment was botched while moving some text around, I think it'd be worthwhile to blame this line and figure out what happened here.

Reply:
I'll follow up outside of this PR for this item